recipe/meta.yaml

```diff
@@ -82,8 +75,9 @@ outputs:
         - pytest
         - scipy
         - pip
-        - jax >= 0.3.24  # [osx or linux64]
+        - jax >=0.4  # [osx or linux64]
         # conda-forge doesn't have pytorch on all platforms/arches yet
```

Comment:
This comment isn't attached to anything anymore, and putting mesalib below makes it even more confusing imo.

Reply:
Done in https://github.com/conda-forge/gymnasium-feedstock/pull/32/commits/1ad951374cbcbba932285429ebd3a3c8ba6a46f0 .